AngouriMath/Core/Antlr/AngouriMath.g

```diff
@@ -136,6 +136,10 @@ atom returns[Entity value]
     | 'limit(' args = function_arguments ')' { Assert("limit", 3, $args.list.Count); $value = MathS.Limit($args.list[0], $args.list[1], $args.list[2]); }
     | 'limitleft(' args = function_arguments ')' { Assert("limitleft", 3, $args.list.Count); $value = MathS.Limit($args.list[0], $args.list[1], $args.list[2], AngouriMath.Core.ApproachFrom.Left); }
     | 'limitright(' args = function_arguments ')' { Assert("limitright", 3, $args.list.Count); $value = MathS.Limit($args.list[0], $args.list[1], $args.list[2], AngouriMath.Core.ApproachFrom.Right); }
+    | 'signum(' args = function_arguments ')' { Assert("signum", 1, $args.list.Count); $value = MathS.Signum($args.list[0]); }
+    | 'sgn(' args = function_arguments ')' { Assert("sgn", 1, $args.list.Count); $value = MathS.Signum($args.list[0]); }
```

Comment:
No `Sgn` for `MathS`?

Reply:
Then it's inconsistent.

Reply:
Functions in MathS don't have synonyms, there's no need for. Say, IDESs will always give you a hint how you should write it. Also, using MathS. functions implies you want something explicit => strict code. While if you parse from a string, you don't want a strict, explicit code, so there're auto-insertable operators, synonyms

Reply:
Well, it's also called the sign function, so why don't you add 'sign(' to here?

Reply:
We can add it I guess